### DISCUSSION TITLE
Windows Defender blocked download of repository zip file

### DISCUSSION THREAD

**Discussion by vlxsqer** [openglobus/openglobus#839] [Q&A]
Hello, not sure if it was better to open this under Issues or here in discussions, but came to post it here in discussions just in case it's something easy to resolve/no need to open an issue for this.

As the title states, when I try to download the repository file from Github, Windows Defender has blocked out the download of the code, stating that it has detected a trojan. Just wanted to check if some part of the code is accidentally flagged as malicious (i.e. false positive), and if that's the case, just see what part of the code could be the reason for that. 

Attached is a screenshot of the protection history from Windows Defender. Thank you!
![image](https://github.com/user-attachments/assets/47c712d2-53af-4da4-8fff-73ae568d8fd1)

**Comment by Zemledelec** [comment]
Trojan in text files, interesting. I will check. Thank you!

**Reply by Zemledelec** [reply]
It looks like a common false positive of antivirus machine learning algorithms. Nevertheless, I will check more carefully.

**Reply by vlxsqer** [reply]
Yea, was suspecting that it might be a common false positive, but just wanted to check here just in case. Do you happen to know which file is triggering the antivirus flag? Thank you so much for helping to look into it. 😓🙏
